### DISCUSSION TITLE
GSOC 2025 Contribution Direction

### DISCUSSION THREAD

**Discussion by AryahP** [MDAnalysis/mdanalysis#4863] [GSoC Discussions]
Hello,

I'm Arya, a first-year Electrical and Computer Engineering student planning on applying for GSOC. I have experience in Python and Javascript, but I am new to open source workflow; I would love it if someone could help direct me to how I can contribute!

Thanks!

**Comment by RMeli** [comment]
Hi @AryahP, 
To familiarise yourself with the MDAnalysis project, I'd strongly suggest to have a deep dive into our User Guide:

https://userguide.mdanalysis.org

Once you are familiar with the project, you can look at "Contributing to MDAnalysis":

https://userguide.mdanalysis.org/stable/contributing.html

However, it should point out that at this stage GSoC 2025 has not been announced yet, and we don't know if we will be selected as an organisation or not.

Best,
Rocco
